### PR TITLE
uploaded unique pairs problem solution in python

### PR DESCRIPTION
**Pull request by Adityanagraj**
please accept my pull request

### CODE REVIEW

**Comment by adad20** [comment]
@Adityanagraj can u add some comments to explain your code

File: week-1/Editorials/Contest 1/problem-3/Unique Number 3.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    for i in arr:
+        if i in dict:
+            dict[i]+=1
+        else:
+            dict[i]=1
+    key_list = list(dict.keys()) 
+    val_list = list(dict.values()) 
+    for i in dict:
+          if dict[i]==1:
+            print(key_list[val_list.index(dict[i])])
```
Comment (adad20) [inline week-1/Editorials/Contest 1/problem-3/Unique Number 3.py:17]: @Adityanagraj Please add some comments to explain the approach

Comment (Adityanagraj) [reply week-1/Editorials/Contest 1/problem-3/Unique Number 3.py]: i have added comments and explained the approach

**Comment by adad20** [comment]
@Adityanagraj Thanks merging this!! Will wait for another too ;)

**Pull request merged**